pyodide-build/pyodide_build/buildall.py

```diff
@@ -470,6 +529,13 @@ def make_parser(parser):
         default=None,
         help=("Only build the specified packages, provided as a comma-separated list"),
     )
+    parser.add_argument(
+        "--force",
```

Comment:
Maybe,
```suggestion
        "--rebuild",
```

to be more explicit ? `buildall --force` is a bit more ambiguous.
Unless we already use it for a similar use case elsewhere?

Reply:
Or maybe `--force-rebuild`?